public/src/client/bugs.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+
+define('forum/bug-report', [
+    'api', 'alerts',
+], function (api, alerts) {
+    const Bugs = {};
+
+    Bugs.init = function () {
+        handleSubmitBug();
+    };
+
+    console.log('hello');
```

Comment:
Not sure if you want to keep it in here, is this just for debugging purposes?

Reply:
Will remove